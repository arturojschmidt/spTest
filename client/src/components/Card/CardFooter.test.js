
import React from 'react';
import { shallow } from 'enzyme';
import CardFooter from './CardFooter.js';

describe('CardFooter', () => {
 it('should be defined', () => {
   expect(CardFooter).toBeDefined();
 });
 it('should render correctly', () => {
   const tree = shallow(
     <CardFooter name='cardfooter test' />
   );
   expect(tree).toMatchSnapshot();
 });
});