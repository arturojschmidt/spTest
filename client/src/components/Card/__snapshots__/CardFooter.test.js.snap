// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardFooter should render correctly 1`] = `ShallowWrapper {}`;
